src/py/flwr/cli/new/new.py

```diff
@@ -58,8 +58,10 @@ def render_template(template: str, data: Dict[str, str]) -> str:
     """Render template."""
     tpl_file = load_template(template)
     tpl = Template(tpl_file)
-    result = tpl.substitute(data)
-    return result
```

Comment:
Whats the reason for adjusting this? Does it fail if there is nothing to substitute?

Reply:
@tanertopal It fails because the `.gitignore` contains a dollar sign

Reply:
Damn. How do we actually deal with files which, contain dollar signs? Maybe we can write the templates differently? Escape them in the template?